rasa/core/validator.py

```diff
@@ -41,17 +43,17 @@ def verify_intents(self, ignore_warnings: bool = True) -> bool:
 
         for intent in self.domain.intents:
             if intent not in nlu_data_intents:
-                logger.warning(
-                    "The intent '{}' is listed in the domain file, but "
-                    "is not found in the NLU training data.".format(intent)
+                warnings.warn(
+                    f"The intent '{intent}' is listed in the domain file, but "
+                    "is not found in the NLU training data."
                 )
                 everything_is_alright = ignore_warnings and everything_is_alright
 
         for intent in nlu_data_intents:
             if intent not in self.domain.intents:
                 logger.error(
```

Comment:
warning?

Reply:
It used to be an error, I though warning is more appropriate

Reply:
Sorry, I meant the `logger.error` should be a `warnings.warn` right?

Reply:
Yes, good catch 👍 there were actually a couple more in there, they are all warnings now